src/main/java/teammates/ui/webapi/UpdateStudentAction.java

```diff
@@ -92,6 +92,7 @@ public JsonResult execute() {
                     return new JsonResult(statusMessage);
                 }
             }
+            taskQueuer.scheduleStudentForSearchIndexing(courseId, studentEmail);
```

Comment:
There's a `resetStudentGoogleId` in the if-block after `updateStudentCascade`, initially I thought we can perform indexing after both actions, but now it seems I can't do that because of the early return. Looks like might need to schedule two tasks, one for `updateStudentCascade` and one for `resetStudentGoogleId`:
```
StudentAttributes updatedStudent = logic.updateStudentCascade(                                     
        StudentAttributes.updateOptionsBuilder(courseId, studentEmail)                             
                .withName(updateRequest.getName())                                                 
                .withNewEmail(updateRequest.getEmail())                                            
                .withTeamName(updateRequest.getTeam())                                             
                .withSectionName(updateRequest.getSection())                                       
                .withComment(updateRequest.getComments())                                          
                .build());                                                                         
taskQueuer.scheduleStudentForSearchIndexing(updatedStudent.getCourse(), updatedStudent.getEmail());
                                                                                                   
if (!student.getEmail().equals(updateRequest.getEmail())) {                                        
    logic.resetStudentGoogleId(updateRequest.getEmail(), courseId);                                
    taskQueuer.scheduleStudentForSearchIndexing(courseId, studentEmail);                                                                                                            
```

Reply:
Google ID is not searchable. Adding a scheduling there is not useful.

Reply:
I was under the impression that admin could search for Google ID in V7?

Reply:
That's instructor. Student google ID is confidential and must not be searchable by instructors, and the search space is the same.

Reply:
Ah, right. Only for instructors.